apps/dashboard/app/(app)/apis/[apiId]/settings/delete-api.tsx

```diff
@@ -68,7 +68,9 @@ export const DeleteApi: React.FC<Props> = ({ api }) => {
     },
     onError(err) {
       console.error(err);
-      toast.error(err.message);
+      let temp = JSON.parse(err.message);
```

Comment:
On second though should `err.message` is all you need. The tRPC error.message will be returned. If you get an error from the DB on the client you are missing an error